profile/README.md

```diff
@@ -0,0 +1,55 @@
+<p align="center">
+<img src="Gen3_Logo-01_blue.png" alt="Gen3 logo" width="500" style="display: block; margin: auto; "/>
+</p>
+
+# Gen3
+
+The University of Chicago, Center for Translational Data Science is the maintainer organization for the open-source Gen3 Data Platform.  Some useful references are included below to help you learn more about Gen3.
+
+## Gen3 community
+
+### Contributing code to Gen3
+
+Gen3 software is completely open-source.  Please refer to our [Contributor Guidelines](https://uc-cdis.github.io/gen3-docs/docs/Contributor%20Guidelines/) for how to contribute code to our repositories.
+
+### Chat with Gen3 staff and the community
+
+If you have questions about operating or using a Gen3 system please contact us.  We can be reached on [slack](https://docs.google.com/forms/d/e/1FAIpQLSczyhhOXeCK9FdVtpQpelOHYnRj1EAq1rwwnm9q6cPAe5a7ug/viewform), [discussion board](https://forums.gen3.org/), or via email (support@datacommons.io).
+
+### Events
+
+We hold regular forums on a variety of topics to engage with and hear from the community.  You find previous and upcomings [here](https://gen3.org/community/events/).
+
+### Gen3 releases
+
+Gen3 releases typically happen every month.  The lastest release notes can be found [here](https://github.com/uc-cdis/cdis-manifest/blob/master/releases/2024/06/gen3-release-notes.md).
+
+## User Guides
+
+### Setting up and operating a Gen3 data commons or mesh
```

Comment:
Since Gen3 on Helm is only for playing and testing, this heading seems slightly misleading?

Reply:
I have updated this phrasing based on a discussion with Jawad.  Helm is now the recommended solution for production.